bemani/format/afp/render.py

```diff
@@ -1099,7 +1099,7 @@ def __place(self, tag: Tag, operating_clip: PlacedClip, prefix: str = "") -> Tup
                 # Get rid of the objects that we're removing from the master list.
                 operating_clip.placed_objects = [
                     obj for obj in operating_clip.placed_objects
-                    if not(obj.object_id == tag.object_id and obj.depth == tag.depth)
+                    if not (obj.object_id == tag.object_id and obj.depth == tag.depth)
```

Comment:
I was like "woah somebody else is working on the AFP renderer?!" and then I was like ....oh.